src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Mvc3/SyncInvokeActionWrapper.cs

```diff
@@ -23,7 +22,7 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
         {
             transaction.AttachToAsync();
 
-            var controllerContext = instrumentedMethodCall.MethodCall.MethodArguments.ExtractNotNullAs<ControllerContext>(0);
+            var controllerContext = instrumentedMethodCall.MethodCall.MethodArguments.ExtractNotNullAs<dynamic>(0);
```

Comment:
Do you want to apply the same `!= null` check to the controllerContext that you did in the AsyncBeginInvokeActionWrapper?

Reply:
done